### ISSUE TITLE
support form with `?` as action

### ISSUE THREAD

**Issue by mayel** [germsvel/phoenix_test#72]
Having a form with `action="?"` should submit to the current path but instead leads to a 404: `no route found for POST /`

**Comment by mayel** [comment]
same with `action="#"`

**Comment by germsvel** [comment]
Yeah, those are interesting. I think those forms of submitting forms to the current path are left over from before the HTML standard made `<form>` _not_ require an `action`. 

But I think we should support that (PhoenixTest itself requires an action, since that's probably what people want). I'd love a PR if you're up for it. Otherwise, I'll try to get that when I can.

**Comment by germsvel** [comment]
@mayel is this still something you're interested in contributing with? If it's not something you need, I'll go ahead and close this issue. If it is, happy to review a PR for this.

**Comment by mayel** [comment]
Yeah I'm usually happy to but don't really have the time at the moment. Also it was quite easy to workaround by changing the link. Maybe good to document at least?

**Comment by germsvel** [comment]
> Also it was quite easy to workaround by changing the link

👍 sounds good. I'm going to close this then. 

> Maybe good to document at least?

I'd be happy to review a PR for docs!
